internal/local/command_kafka_start.go

```diff
@@ -127,79 +130,104 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 	}
 
 	ports := c.Config.LocalPorts
+	platform := &specsv1.Platform{OS: "linux", Architecture: runtime.GOARCH}
 	natKafkaRestPort := nat.Port(ports.KafkaRestPort + "/tcp")
-	natPlaintextPort := nat.Port(ports.PlaintextPort + "/tcp")
-	config := &container.Config{
-		Image:    dockerImageName,
-		Hostname: "broker",
-		Cmd:      strslice.StrSlice{"bash", "-c", "'/etc/confluent/docker/run'"},
-		ExposedPorts: nat.PortSet{
-			natKafkaRestPort: struct{}{},
-			natPlaintextPort: struct{}{},
+	natPlaintextPorts := getNatPlaintextPorts(ports, numOfBrokers)
+	containerStartCmd := strslice.StrSlice{"bash", "-c", "'/etc/confluent/docker/run'"}
+
+	_, err = dockerClient.NetworkCreate(
+		context.Background(),
+		"confluent-local-network",
+		types.NetworkCreate{
+			CheckDuplicate: true,
+			Driver:         "bridge",
 		},
```

Comment:
Can you define this outside of the function call?

Reply:
got it